clrfeatures.cmake

```diff
@@ -25,3 +25,9 @@ endif(NOT DEFINED FEATURE_DBGIPC)
 if(NOT DEFINED FEATURE_INTERPRETER)
   set(FEATURE_INTERPRETER 0)
 endif(NOT DEFINED FEATURE_INTERPRETER)
+
+if(NOT DEFINED FEATURE_NI_BIND_FALLBACK)
```

Comment:
As I understand, the behavior is same if cmakeargs "-DFEATURE_NI_BIND_FALLBACK=..." is absent. Is there another issue that I am aware of?